main.py

```diff
@@ -1,10 +1,52 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as source:
+        data = json.load(source)
+
+    for user in data:
+        race = data[user].get("race")
+        if race and not Race.objects.filter(name=race["name"]).exists():
+            Race.objects.create(
+                name=race["name"],
+                description=race["description"]
+            )
+            skills = race["skills"]
+            if skills:
+                for skill in skills:
+                    if not Skill.objects.filter(name=skill["name"]).exists():
+                        Skill.objects.create(name=skill["name"],
+                                             bonus=skill["bonus"],
+                                             race_id=Race.objects.get(
+                                                 name=race["name"]).id)
+
+        guild = data[user].get("guild")
+        if guild and not Guild.objects.filter(name=guild["name"]).exists():
```

Comment:
try to use get_or_create method to simplify ur code
the same is for similar cases